src/DataGrid.tsx

```diff
@@ -432,6 +438,7 @@ function DataGrid<R, SR, K extends Key>(
     if (checked) {
       newSelectedRows.add(rowKey);
       const previousRowIdx = lastSelectedRowIdx.current;
+      const rowIdx = rows.indexOf(row);
```

Comment:
When we use `indexOf`, should we use `rows` or `rawRows`?

Reply:
We were previously using `const row = rows[rowIdx];` so I guess it should be `rows`